### ISSUE TITLE
[Bug] KubeRay operator can not be deployed

### ISSUE THREAD

**Issue by janessong** [ray-project/kuberay#683]
### Search before asking

- [X] I searched the [issues](https://github.com/ray-project/kuberay/issues) and found no similar issues.

### KubeRay Component

ray-operator

### What happened + What you expected to happen

I cloned the KubeRay GitHub repo and deploy the operator's configuration from my local file system.
use kubectl create -k default/
error: couldn't make loader for namespace.yaml: got file 'namespace.yaml', but '/root/ray/kuberay/ray-operator/config/default/namespace.yaml' must be a directory to be a root

### Reproduction script

kubectl 1.20.9 and k8s cluster was deployed

### Are you willing to submit a PR?

- [ ] Yes I am willing to submit a PR!

**Comment by kevin85421** [comment]
I cannot reproduce this error. My commands are as follows.

```bash
> cd ray-operator/config

> kubectl create -k default
namespace/ray-system created
customresourcedefinition.apiextensions.k8s.io/rayclusters.ray.io created
customresourcedefinition.apiextensions.k8s.io/rayjobs.ray.io created
customresourcedefinition.apiextensions.k8s.io/rayservices.ray.io created
serviceaccount/kuberay-operator created
role.rbac.authorization.k8s.io/kuberay-operator-leader-election created
clusterrole.rbac.authorization.k8s.io/kuberay-operator created
rolebinding.rbac.authorization.k8s.io/kuberay-operator-leader-election created
clusterrolebinding.rbac.authorization.k8s.io/kuberay-operator created
service/kuberay-operator created
deployment.apps/kuberay-operator created
```

This issue https://github.com/kubernetes-sigs/kustomize/issues/3812 might be helpful. In addition, you can also try to upgrade your `kubectl`. My version is as follows.

```bash
> kubectl version
Client Version: version.Info{Major:"1", Minor:"23", GitVersion:"v1.23.8", GitCommit:"a12b886b1da059e0190c54d09c5eab5219dd7acf", GitTreeState:"clean", BuildDate:"2022-06-16T05:57:43Z", GoVersion:"go1.17.11", Compiler:"gc", Platform:"linux/amd64"}
Server Version: version.Info{Major:"1", Minor:"24", GitVersion:"v1.24.0", GitCommit:"4ce5a8954017644c5420bae81d72b09b735c21f0", GitTreeState:"clean", BuildDate:"2022-05-19T15:39:43Z", GoVersion:"go1.18.1", Compiler:"gc", Platform:"linux/amd64"}
```

**Comment by kevin85421** [comment]
This issue is the same as "[Installing KubeRay Operator failing](https://discuss.ray.io/t/installing-kuberay-operator-failing/8204)", and it is solved by upgrading `kubectl` (#702). Closed this issue.

**Comment by janessong** [comment]
> This issue is the same as "[Installing KubeRay Operator failing](https://discuss.ray.io/t/installing-kuberay-operator-failing/8204)", and it is solved by upgrading `kubectl` (#702). Closed this issue.

thanks very much！
